react/features/virtual-background/reducer.js

```diff
@@ -1,9 +1,32 @@
 // @flow
 
-import { ReducerRegistry } from '../base/redux';
+import { PersistenceRegistry, ReducerRegistry } from '../base/redux';
 
 import { BACKGROUND_ENABLED, SET_VIRTUAL_BACKGROUND } from './actionTypes';
 
+/**
+ * The default/initial redux state of the feature {@code features/virtual-background}.
+ *
+ * @type Object
+ */
+const DEFAULT_STATE = {
+    virtualSource: undefined,
+    isVirtualBackground: undefined,
+    backgroundEffectEnabled: undefined
+};
+
+/**
+ * Sets up the persistence of the feature {@code features/virtual-background}.
+ */
+const virtualBackground = {};
+
+// start with the default state
+Object.keys(DEFAULT_STATE).forEach(key => {
+    virtualBackground[key] = true;
+});
+
+PersistenceRegistry.register('features/virtual-background', virtualBackground);
```

Comment:
Since you want all items to be persisted, using `true` instead of the object is simpler.

Reply:
+ Added a const for 'features/virtual-background'